test/white.jl

```diff
@@ -212,8 +212,31 @@ e = [ 5.19346;
     @test pvalue(w_test) ≈ 0.2774 atol=atol
 
     w_test = WhiteTest(X, e)
-    @test pvalue(w_test) ≈ 0.3458 atol=atol
-    show(IOBuffer(), w_test)
+    w_pval = pvalue(w_test)
+    @test w_pval ≈ 0.3458 atol=atol
+
+    refstr = """
+White's (or Breusch-Pagan's) test for heteroskedasticity
```

Comment:
You should be able to add a four-space indentation here without breaking the test.